corehq/form_processor/migrations/0079_migrate_to_bigint.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import RawSQLMigration
+
+
+migrator = RawSQLMigration(('corehq', 'form_processor', 'migrations', 'sql_templates'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('form_processor', '0078_blobmeta_migrated_check'),
+    ]
+
+    operations = [
+        migrator.get_migration('migrate_to_bigint.sql')
```

Comment:
Do you need to add a `SeparateDatabaseAndState` migration here to change the Django column types? As in

https://github.com/dimagi/commcare-hq/blob/e74ae934353e41529387a25a210671733a323356/corehq/blobs/migrations/0003_big_content.py#L23-L30

I think if you don't do that Django will want to create new migrations.

Reply:
Yes, I'm sure you're right. I forgot about that.